.meta/transforms/lua.toml

```diff
@@ -10,6 +10,14 @@ requirements = {}
 
 <%= render("_partials/_component_options.toml", type: "transform", name: "lua") %>
 
+[transforms.lua.options.version]
+type = "string"
+common = true
+required = false
+description = "transform API version"
+default = "1"
+enum = { 1 = "transform API version 1", 2 = "transform API version 2" }
```

Comment:
I would remove `2` until we actually support it. And, at the time, we should make sure it what is presented in the configuration example.

Reply:
Ah, I see, you do have a `v2` here. Does that implement the API discussed in the spec?

Reply:
> Ah, I see, you do have a `v2` here. Does that implement the API discussed in the spec?

It currently a duplicate of `v1`. I'm planning to gradually implement from the RFC in `v2`, so although it is available, the exposed API for it will be very unstable until most of the features are implemented, so it makes sense to hide it until it is somewhat stabilized.